routes/api/users.js

```diff
@@ -71,17 +70,33 @@ router.post("/users/login", function(req, res, next) {
 });
 
 router.post("/users", function(req, res, next) {
-    const user = new User();
+    // const user = new User();
 
-    user.username = req.body.user.username;
-    user.email = req.body.user.email;
-    user.setPassword(req.body.user.password);
+    // user.username = req.body.user.username;
+    // user.email = req.body.user.email;
+    // user.setPassword(req.body.user.password);
+
+    // user.save()
+    //     .then(function() {
+    //         return res.json({ user: user.toAuthJSON() });
+    //     })
+    //     .catch(next);
+
+    const username = req.body.username,
+    email = req.body.email,
+    hash= req.body.password;
+
+    User.create({ username, email, hash})
```

Comment:
Should this be included as part of the task for mongo to postgres migration or should it be a task for another PT story?

Reply:
It's actually not a new feature.  That feature was already implemented in the former setup. I only adapted it to enable me to test the new database setup. You can see I commented the former setup. Thanks for this feedback.